lib/scrabble.rb

```diff
@@ -1,4 +1,104 @@
 module Scrabble
-  # require your gems and classes here
-  # require_relative 'scrabble/whatevs'
+require "awesome_print"
+
+  class Scrabble
+    #___________________________________________________________________________
+    # Constants
+
+    SCORE_1 =  ["A", "E", "I", "O", "U", "L", "N", "R", "S", "T"]
+    SCORE_2 =  ["D", "G"                                        ]
+    SCORE_3 =  ["B", "C", "M", "P"                              ]
+    SCORE_4 =  ["F", "H", "V", "W", "Y"                         ]
+    SCORE_5 =  ["K"                                             ]
+    SCORE_8 =  ["J", "X"                                        ]
+    SCORE_10 = ["Q", "Z"                                        ]
+    #___________________________________________________________________________
+    # Class method score
+
+    def self.score(word)
+
+      # the complex if statement verifies that argument in method:
+      # it's not a number and is not string "0"
```

Comment:
I really like how you document what the conditions are meant to be here.  This is pretty essential because it helps future you and new comers like myself understand your logic.